js/screens/Profile/ProfileContainer.js

```diff
@@ -1,37 +1,45 @@
 import React, { Component } from 'react';
 import Profile from './Profile';
 import PropTypes from 'prop-types';
+import gql from 'graphql-tag';
+import { Query } from 'react-apollo';
+import { Text } from 'react-native';
 
+const CHILD_DATA = gql`
```

Comment:
Stick to the pattern you chose as a group. Either write your query in the `<Query />` component or create a const or just have a file for all of your queries.

Reply:
We are going to create a const for the queries in their respective containers.  I will change the other files to match this one.

Reply:
Thank you!